tensorflow-app-dependencies/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.springframework.cloud.stream.app</groupId>
+    <artifactId>tensorflow-app-dependencies</artifactId>
+    <version>1.0.0.BUILD-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>tensorflow-app-dependencies</name>
+    <description>Spring Cloud Stream tensorflow App Dependencies</description>
+
+    <parent>
+		<artifactId>spring-cloud-dependencies-parent</artifactId>
+        <groupId>org.springframework.cloud</groupId>
+        <version>1.2.1.RELEASE</version>
+        <relativePath />
+    </parent>
+
+    <properties>
+        <pmml.version>1.2.6</pmml.version>
+		<tensorflow.version>1.1.0-rc2</tensorflow.version>
```

Comment:
@tzolov ,

Please, stop confirming the fixes.
Let's treat your silent `Pushed` as an agreement on my comment.
Otherwise your checks look like a spam in my mail box 😉